config/CMakeAddFortranSubdirectory.cmake

```diff
@@ -458,20 +553,46 @@ function( cafs_fix_mpi_library )
 
 endfunction(cafs_fix_mpi_library)
 
-#-------------------------------------------------------------------------------#
+#------------------------------------------------------------------------------#
 # Create imported libraries owned by the Visual Studio project to be used in the
 # CAFS subproject.
-#-------------------------------------------------------------------------------#
+#------------------------------------------------------------------------------#
 function( cafs_create_imported_targets targetName libName targetPath linkLang)
 
-  get_filename_component( pkgloc "${targetPath}" ABSOLUTE )
+  set(verbose_target_name "Lib_foo")
+  if( targetName STREQUAL verbose_target_name )
```

Comment:
This is purely a debugging trick.  To get better debug output from CAFS, I reset this to, say, `Lib_c4_fc4`.  I left all of this debug code in place because this logic is fragile and I seem to be revisiting it every 6 months or so.

Reply:
I really should start a side project to get some of my changes to `CMakeAddFortranSubdirectory.cmake` back into upstream CMake.